lib/elixir/unicode/unicode.ex

```diff
@@ -12,11 +12,25 @@ defmodule String.Unicode do
   @moduledoc false
   def version, do: {7, 0, 0}
 
+  prop_path = Path.join(__DIR__, "PropList.txt")
+  prop_range_line = ~r/\A([0-9A-F]{4})(?:\.\.([0-9A-F]{4}))?/
+  whitespace = Enum.reduce File.stream!(prop_path), [], fn(line, acc) ->
+    case Regex.run(prop_range_line, line) do
+      [_, single] ->
+        [to_binary.(single) | acc]
+      [_, first, last] ->
+        [first_i, last_i] = [String.to_integer(first, 16), String.to_integer(last, 16)]
+        acc ++ Enum.map(first_i..last_i, &(to_binary.(Integer.to_string &1, 16)))
```

Comment:
You can drop the `(` after `&` here (and personally I'd add parens to the `Integer.to_string/2` call :bowtie:):

```elixir
acc ++ Enum.map(first_i..last_i, &to_binary.(Integer.to_string(&1, 16)))
```